lte/gateway/docker/services/s1aptester/Dockerfile

```diff
@@ -0,0 +1,133 @@
+ENV PATH="$VIRTUAL_ENV/bin:$PATH"
+ENV MAGMA_ROOT=/magma
+ENV S1AP_TESTER_ROOT=/s1ap-tester
+ENV S1AP_TESTER_SRC=/S1APTester
+ENV PYTHON_BUILD=/build
+ENV PIP_CACHE_HOME="~/.pipcache"
+ENV TZ=Europe/Paris
+ENV SWAGGER_CODEGEN_JAR=/var/tmp/codegen/modules/swagger-codegen-cli/target/swagger-codegen-cli.jar
+ENV S1AP_TESTER_CFG=$MAGMA_ROOT/lte/gateway/python/integ_tests/data/s1ap_tester_cfg
+ENV GRPC_POLL_STRATEGY=poll
+
+# ENV PYTHON_VERSION=3.8
+
+RUN apt-get update && \
+  apt-get install -y --no-install-recommends \
+  git=$GIT_VERSION \
+  build-essential=$BUILD_ESSENTIAL_VERSION \
+  python3-pip=$PIP_VERSION \
+  python3-venv=$VENV_VERSION \
+  python3-protobuf=$PROTOBUF_VERSION \
+  python3-eventlet=$EVENTLET_VERSION \
+  python3-pystemd=$PYSTEMD_VERSION \
+  libsystemd-dev=$LIBSYSTEMD_DEV_VERSION \
+  virtualenv=$VIRTUALENV_VERSION \
+  lsb-release=$LSB_RELEASE_VERSION \
+  apt-utils=$APT_UTILS_VERSION \
+  sudo=$SUDO_VERSION \
+  libprotobuf-dev=$LIBPROTOBUF_VERSION \
+  openjdk-8-jre-headless=$JRE_VERSION \
+  openjdk-8-jdk=$JDK_VERSION \
+  ruby=$RUBY_VERSION \
+  ruby-dev=$RUBY_DEV_VERSION \
+  wget=$WGET_VERSION \
+  pkg-config=$PKG_CONFIG_VERSION \
+  libmysqlclient-dev=$LIBMYSQL_VERSION \
+  rsync=$RSYNC_VERSION \
+  binutils=$BINUTILS_VERSION \
+  ssh=$SSH_VERSION \
+  sshpass=$SSHPASS_VERSION \
+  libiperf0=$LIBIPERF0_VERSION && \
+  apt-get clean && \
+  gem install fpm
+
+RUN git clone https://github.com/magma/magma $MAGMA_ROOT -b $MAGMA_RELEASE
+
+WORKDIR /var/tmp/
+
+RUN DIRECTORY="/tmp/aioeventlet" && \
+  mkdir -p ${DIRECTORY} && \
+  /magma/third_party/build/bin/aioeventlet_build.sh ${DIRECTORY} && \
+  dpkg -i ${DIRECTORY}/python3-aioeventlet* && \
+  rm -rf ${DIRECTORY}
+
+RUN wget -q http://mirrors.ibiblio.org/apache/maven/maven-3/$MVN_VERSION/binaries/apache-maven-$MVN_VERSION-bin.tar.gz && \
+  tar -xvf apache-maven-$MVN_VERSION-bin.tar.gz && \
+  git clone https://github.com/swagger-api/swagger-codegen /var/tmp/codegen -b $CODEGEN_VERSION
+
+WORKDIR /var/tmp/codegen
+
+RUN ../apache-maven-$MVN_VERSION/bin/mvn clean package -DskipTests
+
+RUN python3 -m venv $VIRTUAL_ENV
+
+WORKDIR /usr/local/bin
+
+RUN ln -s /usr/bin/python3 python
+
+WORKDIR $MAGMA_ROOT/lte/gateway/python
+
+RUN make && \
+  mkdir -p $S1AP_TESTER_ROOT/bin /var/tmp/test_results/
+
+WORKDIR $MAGMA_ROOT/lte/gateway/python/integ_tests
+
+RUN make && \
+  rm -rf /var/lib/apt/lists/* && \
+  $MAGMA_ROOT/lte/gateway/deploy/roles/magma_test/files/clone_s1_tester.sh && \
+  touch /usr/include/stropts.h && \
+  $MAGMA_ROOT/lte/gateway/deploy/roles/magma_test/files/build_s1_tester.sh
+
+RUN pip3 install --cache-dir $PIP_CACHE_HOME pyparsing && \
+  python3 $MAGMA_ROOT/lte/gateway/deploy/roles/magma_test/files/c_parser.py
+RUN sed -i 's/magtivate_cmd + " && " + //g' $MAGMA_ROOT/lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py && \
+  sed -i 's/"vagrant"/"ubuntu"/g' $MAGMA_ROOT/lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py
```

Comment:
Those lines seem like hacky workarounds. Can we include some reasoning here?

Reply:
yes, `s1ap_utils.py` instead should take the user from the env, if present. Or maybe add a CLI option. This also get rid of both `sed` calls.

Reply:
This was not addressed, [afaiks](https://github.com/magma/magma/pull/11308/files#diff-f09d9f258f5ec0d4009c27dd19988e57632fe75dc386bd0180ad083e6da3286eR116-R117).

Reply:
Addressed this in commit https://github.com/magma/magma/pull/11308/commits/b2b253743d202a9a3d5a3007b08af4a5131c8cb4

Instead of deleting the sections where magtivate_cmd is called, I turned the definition into a noop that should always return true fixing it in all cases.